mevolib/fetch/BioSeqs.py

```diff
@@ -550,53 +464,48 @@ def update (self, email):
                                 seq_dict[record.id] = record
                             fetch_handle.close()
                             start += batch_size
-                    # The process has ended correctly so we can replace the old
-                    # dictionary with the new one
+                    # The process has ended correctly so we can replace the old dictionary with the new one
                     self.data = seq_dict
                     # Generate the corresponding report tuple
                     date_time = datetime.now().strftime('%Y/%m/%d %H:%M:%S')
                     self._report.append((date_time, 'entrez', entrez_db, query))
 
-    def write (self, bioseqs_file):
-        """
-        Save all sequences stored at the BioSeqs object in the 'bioseqs_file'
-        (in GENBANK format). A file with a detailed report of the sequences will
-        be created replacing the extension of 'bioseqs_file' by ".rep". If
-        'bioseqs_file' contains a relative path, the current working directory
-        will be used to get the absolute path. If any file already exists, it
-        will be overwritten without warning.
 
-        Arguments:
-            bioseqs_file  (string)
-                New BioSeqs sequence file.
+    def write (self, bioseqs_file: str):
+        """Save all sequences stored at the BioSeqs object in the 'bioseqs_file' (in GENBANK format). A file 
+        with a detailed report of the sequences will be created replacing the extension of 'bioseqs_file' 
+        by ".rep". If 'bioseqs_file' contains a relative path, the current working directory will be used to 
+        get the absolute path. If any file already exists, it will be overwritten without warning.
```

Comment:
```suggestion
        """Saves all sequences stored at the BioSeqs object in the `bioseqs_file` (in GENBANK format).
        
        A file with a detailed report of the sequences will be created replacing the extension of
        `bioseqs_file` by ".rep". If `bioseqs_file` contains a relative path, the current working
        directory will be used to get the absolute path. If any file already exists, it will be
        overwritten without a warning.
```